src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -95,8 +95,8 @@ public function build(ContainerBuilder $container)
         $this->addCompilerPassIfExists($container, AddConsoleCommandPass::class);
         $container->addCompilerPass(new TranslatorPass());
         $container->addCompilerPass(new LoggingTranslatorPass());
-        $container->addCompilerPass(new AddCacheWarmerPass());
-        $container->addCompilerPass(new AddCacheClearerPass());
+        $this->addCompilerPassIfExists($container, AddCacheWarmerPass::class);
```

Comment:
No need for using `addCompilerPassIfExists()` because `http-kernel` is an hard dependency, `addCompilerPass()` should be used directly, that will make the build failure more clear also.